etcdmain/config_test.go

```diff
@@ -15,11 +15,72 @@
 package etcdmain
 
 import (
+	"fmt"
+	"io/ioutil"
+	"log"
 	"net/url"
+	"os"
 	"reflect"
+	"strings"
 	"testing"
+
+	"github.com/ghodss/yaml"
 )
 
+func createCfgFileHelper(b []byte) (*os.File, error) {
```

Comment:
can we move the helper functions to the bottom of the file (this one + validateMemberFlags)?